apps/server/src/modules/user/user.module.ts

```diff
@@ -0,0 +1,12 @@
+/*import { Module } from '@nestjs/common';
+import { TaskController } from './controller/task.controller';
+import { TaskUC } from './uc/task.uc';
+import { TaskRepo } from './repo/task.repo';
+import { SubmissionRepo } from './repo/submission.repo';
+
+@Module({
```

Comment:
when a new module is just not part of the server module it can be tested already